src/components/checkbox/demoSelectAll/script.js

```diff
@@ -0,0 +1,41 @@
+
+angular.module('checkboxDemo3', ['ngMaterial'])
+
+.controller('AppCtrl', function($scope) {
+  $scope.items = [1,2,3,4,5];
+  $scope.selected = [1];
+  $scope.toggle = function (item, list) {
+    var idx = list.indexOf(item);
+    if (idx > -1) list.splice(idx, 1);
+    else list.push(item);
```

Comment:
Yeah I noticed that (this style got Apple in a lot of trouble not too long ago..), this code is pasted from [this file](https://github.com/angular/material/blob/master/src/components/checkbox/demoSyncing/script.js)

Should I can change it in both places?